RecoEgamma/EgammaTools/interface/LongDeps.h

```diff
@@ -0,0 +1,35 @@
+#ifndef LONGDEPS_H
+#define LONGDEPS_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h